tools/harness-thci/README.md

```diff
@@ -57,3 +57,27 @@ Alternatively, platform developers may follow the instructions below to add a ne
 9. Drag the "OpenThread_WpanCtl: Wpantund + NCP" reference device to the **Test Bed** section with the desired number.
 
 10. Select one or more test cases to execute.
+
+
+The above is for the serial connection mode between the DUT and the Host PC. The ethernet connection (only for SSH) mode is
```

Comment:
nit: `The ethernet connection (only for SSH) mode is...` -> `The ssh connection (via ip) is ...`
as wifi should also work in theory.